pipeline/src/main/java/edu/illinois/cs/cogcomp/pipeline/server/ServerClientAnnotator.java

```diff
@@ -110,13 +111,19 @@ public TextAnnotation annotate(String str) throws Exception {
             return SerializationHelper.deserializeTextAnnotationFromBytes(taByte);
         } else {
             URL obj =
-                    new URL(url + ":" + port + "/annotate?text=\""
-                            + URLEncoder.encode(str, "UTF-8") + "\"&views=" + views);
+                    new URL(url + ":" + port + "/annotate");
             HttpURLConnection con = (HttpURLConnection) obj.openConnection();
             con.setRequestMethod("POST");
             con.setRequestProperty("charset", "utf-8");
             con.setRequestProperty("Content-Type", "text/plain; charset=utf-8");
 
+            con.setDoOutput(true);
+            con.setUseCaches(false);
+
+            OutputStreamWriter wr = new OutputStreamWriter(con.getOutputStream());
+            wr.write("text=\"" + URLEncoder.encode(str, "UTF-8") + "\"&views=" + views);
```

Comment:
IIRC, `text=Hello, world&views=POS` is the correct API and not `text="Hello, world"&view=POS`. Current usage  would always add quotes to the text. Missed it in the older PR somehow.